extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1841,10 +1847,12 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {string} full url to safeframe implementation.
    */
   getSafeframePath() {
-    return (
-      'https://tpc.googlesyndication.com/safeframe/' +
-      `${this.safeframeVersion}/html/container.html`
-    );
+    let host = 'tpc.googlesyndication.com';
```

Comment:
nit: const host = isExperimentOn(this.win, 'random-subdomains-for-safeframe') ? (this.safeFrameSubdomain_ + '.safeframe') : 'tpc';

Reply:
done